basyx.submodelrepository-core/src/test/java/org/eclipse/digitaltwin/basyx/submodelrepository/core/SubmodelRepositorySuite.java

```diff
@@ -85,6 +88,37 @@ public void getSpecificNonExistingSubmodel() {
 		repo.getSubmodel("doesNotExist");
 	}
 
+	@Test
+	public void updateExistingSubmodel() {
+		String id = DummySubmodelFactory.SUBMODEL_OPERATIONAL_DATA_ID;
```

Comment:
`DummySubmodelFactory.SUBMODEL_OPERATIONAL_DATA_ID` seems like a bit random here, as it is not clear that one of the submodels from `getSubmodelRepositoryWithDummySubmodels()` has _this_ id.
Suggestion: first create the repo and use the id of a submodel from the repo for testing.